paybybank-us/src/main/java/com/adyen/checkout/paybybankus/PayByBankUSConfiguration.kt

```diff
@@ -8,6 +8,106 @@
 
 package com.adyen.checkout.paybybankus
 
-class PayByBankUSConfiguration {
-    // TODO implement
+import com.adyen.checkout.action.core.GenericActionConfiguration
+import com.adyen.checkout.action.core.internal.ActionHandlingPaymentMethodConfigurationBuilder
+import com.adyen.checkout.components.core.Amount
+import com.adyen.checkout.components.core.AnalyticsConfiguration
+import com.adyen.checkout.components.core.CheckoutConfiguration
+import com.adyen.checkout.components.core.PaymentMethodTypes
+import com.adyen.checkout.components.core.internal.Configuration
+import com.adyen.checkout.components.core.internal.util.CheckoutConfigurationMarker
+import com.adyen.checkout.core.Environment
+import kotlinx.parcelize.Parcelize
+import java.util.Locale
+
+/**
+ * Configuration class for the [PayByBankUSComponent].
+ */
+@Parcelize
+class PayByBankUSConfiguration private constructor(
```

Comment:
Minor: a function should be added to drop in config builder

Reply:
I haven't included the module in drop-in yet, I will do drop-in related parts when I do that. Since it's minor I will leave this one for that PR too.